vsphere/host_virtual_switch_structure.go

```diff
@@ -0,0 +1,151 @@
+package vsphere
+
+import (
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/govmomi/vim25/types"
+)
+
+// schemaHostVirtualSwitchBondBridge returns schema items for resources that
+// need to work with a HostVirtualSwitchBondBridge, such as virtual switches.
+func schemaHostVirtualSwitchBondBridge() map[string]*schema.Schema {
+	return map[string]*schema.Schema{
+		// HostVirtualSwitchBeaconConfig
+		"beacon_interval": &schema.Schema{
+			Type:        schema.TypeInt,
+			Optional:    true,
+			Description: "Determines how often, in seconds, a beacon should be sent to probe for the validity of a link.",
+			Default:     1,
```

Comment:
Unfortunately vSphere doesn't really publish an allowable range, so I don't necessarily want to guess... but I have added a validation for a negative value.